cg/store/crud/update.py

```diff
@@ -71,3 +69,8 @@ def update_sample_sequenced_at(self, internal_id: str, date: datetime):
         sample: Sample = self.get_sample_by_internal_id(internal_id)
         sample.last_sequenced_at = date
         self.session.commit()
+
+    def mark_sample_as_cancelled(self, sample_id: int) -> None:
+        sample: Sample = self.get_sample_by_entry_id(sample_id)
+        sample.is_cancelled = True
```

Comment:
null check? Or do we want the error to be raised? Might be fine-

Reply:
Fixed in https://github.com/Clinical-Genomics/cg/pull/3507/commits/b907fa5abf6fd1c675fe7f57677f3123a1d2fa13
Raised exception